<launch>

    <!-- main launch node -->
    <node pkg="pointgrey_ladybug" type="ladybug_camera" name="ladybug_camera" output="screen" respawn="true">
    <!--<node pkg="pointgrey_ladybug" type="ladybug_camera" name="ladybug_camera" output="screen" launch-prefix="gdb -ex run &#45;&#45;args">-->


        <!-- camera properties -->
        <param name="framerate"               type="double" value="12"/>
        <param name="use_auto_framerate"      type="bool"   value="false"/>
        <param name="shutter_time"            type="double" value="0.8"/>
        <param name="use_auto_shutter_time"   type="bool"   value="false"/>
        <param name="gain_amount"             type="double" value="20"/>
        <param name="use_auto_gain"           type="bool"   value="false"/>

        <!-- post-processing -->
        <param name="jpeg_percent"            type="int"    value="80"/>
        <param name="scale"                   type="double" value="30"/>


    </node>


</launch>
